CoreFoundation/String.subproj/CFString.c

```diff
@@ -5248,9 +5247,7 @@ void CFStringUppercase(CFMutableStringRef string, CFLocaleRef locale) {
 
         if (isEightBit) __CFStringChangeSize(string, CFRangeMake(0, 0), 0, true);
 
-        contents = (UniChar *)__CFStrContents(string);
```

Comment:
Please also revert this change.

Reply:
This is not correctly resolved. It is missing a semicolon and won't compile. There is also an unnecessary whitespace change.